scripts/smac-validate.py

```diff
@@ -75,15 +79,29 @@
                               cost_for_crash=scenario.cost_for_crash)
     if args_.tae == "aclib":
         tae = ExecuteTARunAClib(ta=scenario.ta,
-                              run_obj=scenario.run_obj,
-                              par_factor=scenario.par_factor,
-                              cost_for_crash=scenario.cost_for_crash)
+                                run_obj=scenario.run_obj,
+                                par_factor=scenario.par_factor,
+                                cost_for_crash=scenario.cost_for_crash)
 
     validator = Validator(scenario, trajectory, args_.output,
                           args_.seed)
-    validator.validate(config_mode=args_.configs,
-                       instance_mode=args_.instances,
-                       repetitions=args_.repetitions,
-                       n_jobs=args_.n_jobs,
-                       runhistory=args_.runhistory,
-                       tae=tae)
+
+    # Load runhistory
+    if args_.runhistory:
+        runhistory = RunHistory(average_cost)
```

Comment:
Can we use only a single runhistory or could we use also use all runhistories from independent runs?

Reply:
Now we can use multiple ones by using `--runhistory path_to_rh_1 path_to_rh_2 path_to_rh_3` and so on.